commcare_connect/opportunity/visit_import.py

```diff
@@ -77,14 +85,20 @@ def get_missing_message(self):
         return f"<br>{len(self.missing_completed_works)} completed works were not found:<br>{'<br>'.join(missing)}"
 
 
+@dataclass
+class CatchmentAreaImportStatus:
+    seen_catchments: set[str]
+    missing_catchments: int
+
+    def __len__(self):
+        return len(self.seen_catchments)
+
+    def get_missing_message(self):
+        return f"<br>{self.missing_catchments} catchment areas were not found."
```

Comment:
This makes it seem like there is an error when one is missing, but that is the only way to create new ones, right? I think this can be reworded to say something like "X catchment areas created" and the variable can be renamed `new_catchments` to be clear its not a problem.

Reply:
fixed